lib/routes/token.js

```diff
@@ -29,11 +29,19 @@ const GRANT_JWT = 'urn:ietf:params:oauth:grant-type:jwt-bearer';
 const JWT_AUD = config.get('publicUrl') + '/v1/token';
 
 const SERVICE_CLIENTS = {};
-const JWTOOL = new JwTool(config.get('serviceClients').map(function(client) {
+const SERVICE_JWTOOL = new JwTool(config.get('serviceClients').map(function(client) {
   SERVICE_CLIENTS[client.jku] = client;
   return client.jku;
 }));
 
+const SCOPE_OPENID = 'openid';
+
+const ID_TOKEN_EXPIRATION = Math.floor(config.get('openid.ttl') / 1000);
+const ID_TOKEN_ISSUER = config.get('openid.issuer');
+const ID_TOKEN_KEY = JwTool.JWK.fromObject(config.get('openid.key'), {
+  iss: ID_TOKEN_ISSUER
+});
+
```

Comment:
I wonder if we should also say a few words about this in the API docs for this endpoint.  We don't need to explain what OIDC *is*, but we can broadly say that this endpoint is compatible with OIDC code flow or something similar.